.github/workflows/buildandpublish.yml

```
@@ -0,0 +1,31 @@
+name: Build and publish Ansible
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build-and-publish:
+    name: Build and publish Ansible to PyPI
+    runs-on: ubuntu-latest
```

Make sure to use an environment.
```suggestion
    runs-on: ubuntu-latest

    environment:
      name: pypi
      url: >-
        https://pypi.org/project/ansible-core

```
Also, create the `PYPI_API_TOKEN` secret under that `pypi` environment.
Moreover, add a required approval there and maybe a cooldown timer.